src/components/search-box.vue

```diff
@@ -1,23 +1,36 @@
 <template>
-  <div
-    class="x-input-group"
-    :class="$x.device === 'mobile' ? 'x-input-group--card' : 'x-input-group--line'"
-  >
-    <SearchIcon />
-    <SearchInput :placeholder="$t('searchBox.placeholder')" />
-    <ClearSearchInput>{{ $t('searchBox.clear') }}</ClearSearchInput>
+  <div class="x-input-group x-input-group--card">
```

Comment:
The border color should be the lead one, I think. 
```suggestion
  <div class="x-input-group x-input-group--card x-border-color--lead">
```

Reply:
yes, you are right